include/pixmapstr.h

```diff
@@ -97,23 +97,13 @@ typedef struct _PixmapDirtyUpdate {
     struct pixman_f_transform f_transform, f_inverse;
 } PixmapDirtyUpdateRec;
 
-static inline void
-PixmapBox(BoxPtr box, PixmapPtr pixmap)
-{
-    box->x1 = 0;
-    box->x2 = pixmap->drawable.width;
-
-    box->y1 = 0;
-    box->y2 = pixmap->drawable.height;
-}
-
-
 static inline void
 PixmapRegionInit(RegionPtr region, PixmapPtr pixmap)
 {
-    BoxRec box;
-
-    PixmapBox(&box, pixmap);
+    BoxRec box = {
+        .x2 = (int16_t)pixmap->drawable.width,
+        .y2 = (int16_t)pixmap->drawable.height,
+    };
```

Comment:
It's kind of obvious, but I would still add a comment like "defining a (0, 0) - (drawable.width, drawable.height) box" or something like that.